### PR TITLE
Update the getting started section

### PR DESCRIPTION
**Pull request by irfanpena**
## Describe Your Changes

- Overview
- Installation
- CLI command
- Server

## Fixes Issues

- Closes #
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [x] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: docs/installation/linux.mdx [hunk 1]
```diff
@@ -12,96 +12,23 @@ import Admonition from '@theme/Admonition';
...
-
-:::info
-You can also install Cortex using the Cortex Installer available on [GitHub Releases](https://github.com/janhq/cortex/releases).
-:::
+Before installation, make sure that you have met the required [dependencies](#dependencies) and [hardware](#hardware) to run Cortex.
 
 ```sh
-# Install globally on your system
+# Install OpenMPI dependency
+sudo apt install openmpi-bin libopenmpi-dev
```
Comment (gabrielle-ong) [inline docs/installation/linux.mdx:20]: Is this needed in all cases? I didnt seem to need to run this

File: docs/installation/windows.mdx [hunk 2]
```diff
@@ -12,119 +12,34 @@ import Admonition from '@theme/Admonition';
...
-
-## Cortex Installation
-
-### Step 1: Install Cortex
+## Overview
+For Windows, Cortex can be installed either directly on **Windows** or on **Windows Subsystem for Linux (WSL)**.
 
 :::info
-You can also install Cortex using the Cortex Installer available on [GitHub Releases](https://github.com/janhq/cortex/releases).
+If you're unsure about what is **Windows Subsystem for Linux (WSL)**, please refer to this [article](https://learn.microsoft.com/en-us/windows/wsl/faq).
```
Comment (gabrielle-ong) [inline docs/installation/windows.mdx:19]: Just link within the previous statement? To have one less one info box (less clutter)

File: docs/installation/windows.mdx [hunk 3]
```diff
@@ -12,119 +12,34 @@ import Admonition from '@theme/Admonition';
...
-:::
+Before installation, make sure that you have met the required [dependencies](#windows-subsystem-for-linux) and [hardware](#hardware) to run Cortex.
 
 ```sh
-# Install globally on your system
+# Install using NPM
 npm i -g cortexso
 
 # Install using Brew
 brew install cortexso
```
Comment (gabrielle-ong) [inline docs/installation/windows.mdx:37]: Cmiiw, brew cant be used for windows (Im coming from Mac Land), so lets remove brew? I also dont see Brew in the original notion for windows
https://www.notion.so/jan-ai/Windows-b85b40faefbb40eea2f81f2716f4cc9a?pvs=4

File: docs/installation/windows.mdx [hunk 4]
```diff
@@ -155,9 +70,58 @@ cortex -h
...
+
+## Prerequisites
+### Dependencies
+#### Windows
+- Node.js version 18 and higher
+- NPM
+#### Windows Subsystem for Linux
+- Node.js version 18 and higher
+- NPM
+- Homebrew
```
Comment (gabrielle-ong) [inline docs/installation/windows.mdx:91]: minus brew

File: docs/installation/windows.mdx [hunk 5]
```diff
@@ -155,9 +70,58 @@ cortex -h
 
 ## Uninstall Cortex
 ```sh
-# Uninstall globally on your system
+# Uninstall using NPM
 npm uninstall -g cortexso
 
 # Uninstall using Brew
 brew uninstall cortexso
```
Comment (gabrielle-ong) [inline docs/installation/windows.mdx:77]: remove brew

**Review by gabrielle-ong — APPROVED** [review]
Just the 2 changes: 1/ remove Brew install/uninstall/dependencies in windows~, 2/ remove warning box for WSL then LGTM! 
(edit: nvm about 1, I understand now it is for WSL acting like Linux)

File: docs/basic-usage/command-line.md [hunk 6]
```diff
@@ -8,25 +8,40 @@ slug: "command-line"
...
+### Show the Model State 
+```bash
+# Show a model and cortex system status
 cortex ps
-# Stops a model
+```
+### Stop Model
+```bash
+# Stop a model
 cortex stop
```
Comment (gabrielle-ong) [inline docs/basic-usage/command-line.md:41]: Change to `cortex kill`.  the current syntex deployed is still cortex kill (supposed to be cortex stop in the future)

**Review by urmauur — APPROVED** [review]

**Pull request merged**